lib/command.c

```diff
@@ -1213,7 +1213,9 @@ static int handle_pipe_action(struct vty *vty, const char *cmd_in,
 			goto fail;
 		}
 		*cmd_out = XSTRDUP(MTYPE_TMP, cmd_in);
-		*(strstr(*cmd_out, "|")) = '\0';
+
+		char *u = *cmd_out;
```

Comment:
`check_path.pl` is complaining that there is a missing newline here. Either (a) move the `char *u` declaration to the top of the block/function or (b) just add a new line.

Reply:
Thank you, Rafael. I'll use check_patch.pl in next fixes. I'll push the fix for this in few minutes.